### ISSUE TITLE
[Bug]: Example for cypress_module_test gives false positives.

### ISSUE THREAD

**Issue by mikelikespie** [aspect-build/rules_cypress#44]
### What happened?

Not actually a bug, but error in docs (and `tests/module_test/runner.js`)
I was following the example snippet for runner for `cypress_module_test` and noticed it gave false positives when a test failed but nothing else.
If a test failed, result.status === 'finished' and one has to check the failed count, so in addition to checking status, one has to do something like

``` 
 if (result.failures) {
      console.error('Some tests failed')
      console.error(result.message)
      process.exit(1)
    }
```

### Version

Development (host) and target OS/architectures:
macos arm

Output of `bazel --version`:
bazel 6.2.0

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

`v0.2.0` of rules_cypress

cypress_version = "12.3.0",

### How to reproduce

```shell
Add a failing cypress test to example, and notice it doesn't fail.
```

**Comment by jbedard** [comment]
Supposedly fixed by #51
